pkg/gitapi/gitapi.go

```diff
@@ -12,7 +12,14 @@ import (
 )
 
 func GetList(config *config.Config) []string {
-	req, err := http.NewRequest(http.MethodGet, "https://api.github.com/orgs/"+config.OrgaName+"repos", nil)
+	option := ""
+	if config.OrgaRepoType != "" {
+		option = "type=" + config.OrgaRepoType
+		logr.Infof("[API] Using type option: %v", option)
+	}
+
+	url := "https://api.github.com/orgs/" + config.OrgaName + "repos?" + option
```

Comment:
Use `url.JoinPath` and other functions from the package to create the URL. This minimises the chance of errors caused by malformed URLs.

Reply:
Resolved in 249b469